nagoya.html

```diff
@@ -217,87 +216,85 @@ <h3>インフォメーション</h3>
           <!-- Feel free to group the sponsors as you wish. E.g media, friends, premium sponsors etc. You know best the situation in your city! Pictures should be either 100 x 100px or 250 x 90px -->
           <hr />
           <h3>パートナー</h3>
-            <p>Rails Girls Nagoya は以下のすばらしいパートナーとの共同開催です。<br>
-             Rails Girls Nagoyaではご支援いただけるパートナーを募集しております。 <a href="https://docs.google.com/forms/d/1LZ8vMxJo5N0s-vN6vkNiZKf9Eipy9A6u6zaQlP1z-t4/edit">こちら</a>までご連絡ください！</p>
-
-          <p>Rails Girls Nagoya は以下のすばらしいパートナーとの共同開催です。<br>
+          <p>
+            ご支援いただけるパートナーを募集しております。<br>
+            <a href="https://forms.gle/hEc8NTUfZDXCGNGL6" target="_blank">こちらのフォーム</a>からお問合せください。
```

Comment:
this link is tokyo 16th's link.
please fix your nagoya 6th's link.

Reply:
fix